Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -523,14 +523,21 @@ static inline void continueAfterRequest(RetainPtr<id <ASCCredentialProtocol>> cr
     }).get()];
 }
 
-void WebAuthenticatorCoordinatorProxy::isConditionalMediationAvailable(QueryCompletionHandler&& handler)
+static inline bool isConditionalMediationAvailableImpl(const WebCore::SecurityOriginData& data)
```

Comment:
nit: This name strikes me as odd. Maybe we can name it similar to the SPI `canCurrentProcessAccessPasskeysForRelyingParty`?

Reply:
Done